.github/workflows/lint.yml

```diff
@@ -213,3 +213,38 @@ jobs:
           git ls-files -z -- bootstrap '*.cmake' '*.cmake.in' '*CMakeLists.txt' | \
           grep -E -z -v '^(cmake/Modules/|cmake/Modules_CUDA_fix/)' | \
           xargs -0 cmakelint --config=.cmakelintrc --spaces=2 --quiet
+
+  mypy:
+    runs-on: ubuntu-18.04
+    steps:
+      - name: Setup Python
+        uses: actions/setup-python@v1
+        with:
+          python-version: 3.8
+          architecture: x64
+      - name: Fetch PyTorch
+        uses: actions/checkout@v2
+        with:
+          ref: ${{ github.event.pull_request.head.sha }}
+      - name: Get HEAD commit SHA
+        run: echo ::set-output name=commit-sha::$(git rev-parse HEAD)
+        id: get-commit-sha
```

Comment:
This step is unnecessary:
```suggestion
```

Reply:
Will do in followup (to avoid waiting once again for internal CI)